CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+## Generating Mocks
+
+As of writing, the mock objects are generated in this project's `mock` directory. When a given component is altered and its test suite depends upon a mock object new mocks need to be generated in the `mock` directory to replace the now outdated mocks. 
+
+To generate new mocks, while in this project's root directory execute this command:
+
+```shell
+go generate ./...
+```
```

Comment:
I know it seems prudent to just generate them in CI. My concern was testing on your local machine. I did not come up with a way of facilitating that. 

'We' is a generous term. Since no one else is really developing this project, it is predominantly a note to myself.